packages/core/admin/admin/src/content-manager/hooks/useFetchContentTypeLayout/utils/tests/formatLayouts.test.js

```diff
@@ -341,7 +341,7 @@ describe('Content Manager | hooks | useFetchContentTypeLayout | utils ', () => {
   });
 
   describe('formatLayoutWithMetas', () => {
-    it('should return a layout with the metadas for each input', () => {
+    it('should return a layout with the metadata for each input', () => {
```

Comment:
I'm not really sure about this. Maybe it meant `metas`?
Or maybe the `describe` should be `Metadata`?

Reply:
The test is correct here.